packages/clients/textLocator/src/mapConfig.ts

```diff
@@ -0,0 +1,109 @@
+// number-only keys needed in layers object
+/* eslint-disable @typescript-eslint/naming-convention */
+import { MapConfig } from '@polar/lib-custom-types'
+import locales from './locales'
+import {
+  openStreetMap,
+  openSeaMap,
+  mdiSeaNames,
+  wmtsTopplusOpenWeb,
+  wmtsTopplusOpenWebGrey,
+  wmtsTopplusOpenLight,
+  wmtsTopplusOpenLightGrey,
+  aerial,
+} from './services'
+import { theme } from './palettes'
+
+let zoomLevel = 0
+
+// NOTE: Only configure core properties here; plugin details in `addPlugins.ts`
+export const mapConfiguration: Partial<MapConfig> = {
```

Comment:
The solution seems to be overly complex for the current situation that could, to the same effect, be changed to `Pick<MapConfig, 'list' | 'all' | 'the' | 'keys'>` and then making a copy of the object in `polar-client.ts` with the complete type. What would be the advantage of this remodeling?

`layerConf` is set over there since it's at hand for the `initializeLayerList` anyway.

Reply:
Go for the ticket.